.eslintrc.yml

```diff
@@ -21,7 +21,7 @@
       - 2
       - "1tbs"
     callback-return: 2
-    camelcase: 2
+    camelcase: 0
```

Comment:
Oh my!

Reply:
i wanted camelcase for "must_not"....

Reply:
oh, actually i see how you did it for `ne`. i'll change it